portfolio/src/main/webapp/index.html

```diff
@@ -17,14 +17,11 @@ <h2>Click here for a random fact about me:</h2>
             <div id="greeting-container"></div>
       </div>
       
-      <form action="/data" method="POST">
-            <p>What is a fun fact about you? </p>
-            <textarea name="text-input"> type here... </textarea>
-            <br/></br>
-            <input type="submit" name="submit" value="submit" >
+      <form>
+        <input type="text" id="text-area"/>
+        <input type="button" value="submit" onclick="getServerName()"/>
```

Comment:
The onclick function name is a bit misleading. The `get` prefix signifies that you are getting a value from somewhere. In this case, you should name it something like `addFunFact()` or something similar.